ouroboros-consensus/src/ouroboros-consensus/Ouroboros/Consensus/HardFork/Simple.hs

```diff
@@ -42,12 +42,12 @@ instance SerialiseNodeToClient blk TriggerHardFork where
   encodeNodeToClient _ _ triggerHardFork = case triggerHardFork of
     TriggerHardForkAtVersion v -> encodeListLen 2 <> encodeWord8 0 <> toCBOR v
     TriggerHardForkAtEpoch e -> encodeListLen 2 <> encodeWord8 1 <> toCBOR e
-    TriggerHardForkNotDuringThisExecution -> encodeListLen 2 <> encodeWord8 2
+    TriggerHardForkNotDuringThisExecution -> encodeListLen 1 <> encodeWord8 2
```

Comment:
Just confirming: Doing this fix without special backwards-compat dances is fine because it is only used for `DebugLedgerConfig`?

Reply:
Indeed :+1: